.flowconfig

```diff
@@ -1,4 +1,5 @@
 [options]
+types_first=false
```

Comment:
unfortunately, classic-mode was removed in the 0.143 release of `flow-bin`:
https://github.com/facebook/flow/releases/tag/v0.143.0

enabling the `types_first` mode leads to over 1800 errors are the moment. i am afraid we cannot update to version higher than 0.142 without spending a lot of time on this. (looking at the popularity of flow, i feel like this time would be better spent on migrating to typescript 🙈 )